src/styles/searchResults.css

```diff
@@ -0,0 +1,35 @@
+.search-results {
+  align-items: center;
```

Comment:
This property isnt doing anything so can be removed

Reply:
Removed.